python/semantic_kernel/connectors/memory/qdrant/qdrant_memory_store.py

```diff
@@ -0,0 +1,385 @@
+# Copyright (c) Microsoft. All rights reserved.
```

Comment:
Can we add an example / unit test for this? Just so we can see it running. Or if you're planning to do that with all the other memory stores that are coming for Python, that's works too!

Reply:
@alexchaomander We discussed config settings for the vector memorystores. Should I remove the config file in the interim?

Reply:
@tawalke yeah I would remove the separate config file. It should just be a direct part of instantiating the MemoryStore

Can we at least have a super simple unit test? Just an import and using this memory store in a kernel. We can save embedding real data for the notebook example.

Reply:
@alexchaomander removed config in https://github.com/microsoft/semantic-kernel/pull/919/commits/e062fa799c4a38153cc809cf84f15c250977301f 

Base unit test created will enhance.